src/deploy.go

```diff
@@ -278,10 +325,13 @@ func renderHTML(options Options, file HTMLFile) string {
 						break
 					}
 				}
+				// TODO(renandincer): take a second look here
```

Comment:
The odds are you forget this is here.  It might be a better item for your personal todo list.